membership-attribute-service/app/wiring/AppLoader.scala

```diff
@@ -57,21 +57,15 @@ class MyComponents(context: Context)
     "/user-attributes/me/mma-membership",
     "/user-attributes/me/mma-paper")
 
-  val mmaUpdatePaths = Seq(
-    "/user-attributes/me/membership-update-card",
-    "/user-attributes/me/digitalpack-update-card",
-    "/user-attributes/me/paper-update-card",
-    "/user-attributes/me/contribution-update-card",
-    "/user-attributes/me/cancel-regular-contribution",
-    "/user-attributes/me/contribution-update-amount")
+  val postPaths = router.documentation.filter(triple => triple._1.contains("POST")).map(triple => triple._2)
```

Comment:
we try to avoid using _1 and _2 and use case (some,somethingElse) instead to make the code more readable. In this case you can probably also replace the map and filter with a collect.
maybe you can check if something like this works:
```scala
val postPaths = router.documentation.collect { case ("POST", path, _) => path }
```
in any case I'm not sure if using the generated Routes like this is something we should avoid or not, maybe someone who knows more about play can give their opinion as well